<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSettingOriginsForm</class>
 <widget class="QWidget" name="QSettingOriginsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_pgup">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pgdn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_page">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_page">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="HNMPTableWidget" name="tb_settingOrigins_list" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>451</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNMPTableWidget</class>
   <extends>QWidget</extends>
   <header location="global">hnmptablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
